src/ai/commands/parsers/fast_transcription_command_parser.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.CommonForCli.Details.Commands.Parsers;
+using Microsoft.CommonForCli.Details.NamedValues;
+using Microsoft.CommonForCli.Details.NamedValues.Parsers;
+
+namespace Microsoft.AI.CLI.Commands.Parsers
+{
+    public class FastTranscriptionCommandParser : CommandParser
+    {
+        public FastTranscriptionCommandParser()
+        {
+            AddTokenParser(new RequiredValidValueNamedValueTokenParser("key", "The subscription key for Azure Speech service"));
```

Comment:
That's not how `RequiredValidValueNamedValueTokenParser` works... 

Nor later in this file is how `OptionalValidValueNamedValueTokenParser` works, nor `TrueFalseNamedValueTokenParser`